OpenAI.NET.Tests.Acceptance/Clients/Completions/CompletionClientTests.Prompts.cs

```diff
@@ -30,8 +29,8 @@ public async Task ShouldPromptCompletionAsync()
             ExternalCompletionResponse completionResponse =
                 CreateRandomExternalCompletionResponse();
 
-            Completion expectedCompletion = inputCompletion.DeepClone();
-            expectedCompletion = ConvertToCompletion(inputCompletion, completionResponse);
+            Completion expectedCompletion =
+                ConvertToCompletion(inputCompletion, completionResponse);
```

Comment:
Hi, @Gummimundur, sorry about the intrusion, but I think that this wasn't what was intended for the test. 

The method `ConvertToCompletion` actually updates the parameter `inputCompletion` and also returns it, so doing it like this will make `expectedCompletion` and `inputCompletion` the same object. 
Which means that when we do this:
```
  Completion actualCompletion =
                await this.openAIClient.Completions.PromptCompletionAsync(
                    inputCompletion);
```
`actualCompletion` will also be the same object as `expectedCompletion` and `inputCompletion`, so the test will always pass, even if `PromptCompletionAsync` method does nothing,

I suggest doing it like this, so that the `expectedCompletion` and the `inputCompletion` objects are not the same. Although, technically it is not necessary, I would keep the attribution (`expectedCompletion = ConvertToCompletion`) because it makes it easier to understand.

```suggestion
            Completion expectedCompletion = inputCompletion.DeepClone();
            expectedCompletion = ConvertToCompletion(expectedCompletion, completionResponse);
```

Reply:
Hi @diogod3, I appreciate the feedback. You are absolutely right. Well spotted!
This CODE RUB is irrelevant now therefore this PR should probably be closed without merge?
And you can open another PR with your suggestion?